drivers/jason0x43-wemo_switch.groovy

```diff
@@ -179,3 +188,84 @@ private debugLog(message) {
 private syncTime() {
     parent.childSyncTime(device)
 }
+
+
+def updatePort(port) {
+    iPort=s2i(port)
```

Comment:
If the port is a number, it wouldn't need to be converted here. Also, `iPort` needs a `def`.